cse-cc-import-runner-app/src/main/java/com/farao_community/farao/cse/import_runner/app/services/CseRunner.java

```diff
@@ -76,7 +75,6 @@ public CseRunner(FileImporter fileImporter, FileExporter fileExporter, MultipleD
         this.initialShiftService = initialShiftService;
     }
 
-    @Threadable
```

Comment:
Is the Threadable class still used somewhere else in the code? Or can we delete it?